api/src/controllers/young/inscription.js

```diff
@@ -731,4 +731,37 @@ router.put("/done", passport.authenticate("young", { session: false, failWithErr
   }
 });
 
+router.put("/save/:type", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
```

Comment:
j'aurais plutot fait une autre route perso.
mais sinon on peut rendre le parametre optionnel :
`/save`
`/save/correction`

```suggestion
router.put("/save/:type?", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
```